### PR TITLE
new RPC for all public keys CORE-3065

### PR DESCRIPTION
**Pull request by mmaxim**
@maxtaco this still needs a test, but I wanted to check to see if this is on the right track first.

### CODE REVIEW

File: go/service/user.go [hunk 1]
```diff
@@ -129,3 +130,18 @@ func (h *UserHandler) LoadPublicKeys(_ context.Context, arg keybase1.LoadPublicK
...
+func (h *UserHandler) LoadAllPublicKeysUnverified(_ context.Context,
+	arg keybase1.LoadAllPublicKeysUnverifiedArg) (keys []keybase1.PublicKey, err error) {
+
+	body := &jsonw.Wrapper{}
+	u, err := libkb.LoadUserFromServer(h.G(), arg.Uid, body)
+	if err != nil {
+		return
+	}
+	var publicKeys []keybase1.PublicKey
+	if u.GetKeyFamily() != nil {
```
Comment (maxtaco) [inline go/service/user.go]: Won't this load just the most recent key family?

**Comment by maxtaco** [comment]
As for the test, you might add a new `t_tracy`-style user to the DB, call it `t_rosetta`, let's say. That is:

* sign up and add keys for `t_rosetta`
* reset `t_rosetta`'s sigchain
* add more keys
* export `t_rosetta` to `keybase.sql` and to a patch file (@oconnor663 made a script do this)

Then, in your test, make sure you're getting the old and the new keys for `t_rosetta`

File: go/service/user.go (hunk 1 above)
Comment (mmaxim) [inline go/service/user.go]: It appears to be all of them:

       if @public_keys.all?
          res.public_keys or= {}
          # This includes old versions of PGP keys that may be revoked or
          # superseded.
          res.public_keys.all_bundles = (key.bundle for key in @public_keys.all)

Comment (maxtaco) [inline go/service/user.go]: Then `GetKeyFamily` is pretty badly named....

Comment (mmaxim) [inline go/service/user.go]: And this:

// As returned by user/lookup.json
type RawKeyFamily struct {
	AllBundles []string `json:"all_bundles"`
}

**Comment by mmaxim** [comment]
Test in play!

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **Folders_Ignore_Confirmation-Normal_Private** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1465932933213-f5be895f961c-ebbef8dd7c7e/Folders_Ignore_Confirmation-Normal_Private-f5be895f961c.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1465932933213-f5be895f961c-ebbef8dd7c7e/Folders_Ignore_Confirmation-Normal_Private-ebbef8dd7c7e.png)  
   Folders_Ignore_Confirmation-Normal_Private comparison

**Comment by maxtaco** [comment]
👍

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **Search_Non_User_Pane-No_Avatar** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466002891281-92757bafc3dd-f5b3300e9a0c/Search_Non_User_Pane-No_Avatar-92757bafc3dd.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466002891281-92757bafc3dd-f5b3300e9a0c/Search_Non_User_Pane-No_Avatar-f5b3300e9a0c.png)  
   Search_Non_User_Pane-No_Avatar comparison

**Pull request merged**